src/components/atoms/button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';

```

Comment:
I think it's better to move all of the test files to another directory called `tests`. This can help to prevent having lots of files in one directory.